scripts/deployments/deploy_everything_locally.bash

```diff
@@ -235,6 +235,9 @@ if [ "$start_opsstack" -eq 0 ]; then
     # -------------------------------- ADMIN-PANELS -------------------------------
 
     log_info "starting admin-panels..."
+    # Check if the stack 'admin-panels' exists and delete it if it does
+    # shellcheck disable=2015
+    docker stack ls | grep -q admin-panels && docker stack rm admin-panels >/dev/null 2>&1 || true
```

Comment:
Why do we need to delete `admin-panels` specifically before starting it while all other services in this script are simply started?

Is there some specific corner case?

Reply:
Yes, somehow the jinja2-templating logic that automatically adds jupyter-notebooks that it finds in a folder as `docker configs`. Docker always detects the docker config objects as changed and complains that he cant update the service if docker configs change. This adresses it in a bit shitty manner (always cleanly recreating), but it should be fine for the `osparc.local` usecase I think.

It fixes an issue and adds robustness, at the cost of being a bit slower by first deleting and then recreating, but only on the local deploy.

Reply:
But why does not it happen in running deployments?

Reply:
I think it is related to the `jupyter_server_config.py` on my machine somehow. I have not seen it in deployments, only on my local machine. Might be again some custom ubuntu tooling, where I have the GNU version, or something along those lines?